<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QNewProjectDialog</class>
 <widget class="QDialog" name="QNewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="projectNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="levelGroupBox">
     <property name="title">
      <string>First level</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="levelNameLabel">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="levelNameEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="levelWidthLabel">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="levelWidthSpin">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string> tiles</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="levelHeightLabel">
          <property name="text">
           <string>Height</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="levelHeightSpin">
          <property name="suffix">
           <string> tiles</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="tileSizeLabel">
          <property name="text">
           <string>Tile size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="tileSizeSpin">
          <property name="minimumSize">
           <size>
            <width>77</width>
            <height>0</height>
           </size>
          </property>
          <property name="suffix">
           <string> px</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>layoutWidget_2</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QNewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QNewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>109</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
